### DISCUSSION TITLE
windows ink not working

### DISCUSSION THREAD

**Discussion by IcedMochaTea** [OpenTabletDriver/OpenTabletDriver#1239] [Q&A] [answered]
ive gotten the drivers because i wanted to play osu and draw but windows ink pugin doesnt pick up my pen
if i use the normal absolute mode everything works, but when im on any windows ink mode it just doesnt want to move my pen

**Comment by jamesbt365** [comment]
Long wait but you are missing vmulti driver, shown here: https://github.com/X9VoiD/vmulti-bin/releases/tag/v1.0

**Comment by gonX** [comment, marked as answer]
Please read the WindowsInk wiki carefully: https://github.com/X9VoiD/VoiDPlugins/wiki/WindowsInk
